pandas/core/indexes/datetimes.py

```diff
@@ -2128,19 +2128,34 @@ def month_name(self, locale=None):
 
     def day_name(self, locale=None):
         """
-        Return the day names of the DateTimeIndex with specified locale.
+        Return the day names.
+        
+        Provides day names of the DateTimeIndex for the dataframe which consists of the time and date data with specified locale.
 
         Parameters
         ----------
         locale : string, default None (English locale)
-            locale determining the language in which to return the day name
-
+            The locale determines the language in which the day name should be returned.
```

Comment:
This should go just below the `locale` line.

Reply:
Mention that the default is the current locale, which can be obtained with `locale.getlocale()`.